### PR TITLE
#102 button adjustments

### LINKED ISSUE #98: Bug - Major - Textual content with images - Button is not correctly wrapped on Mobile view
The button from the Textual content with images is not correctly wrapped on Mobile view.

**Steps to reproduce:**
1. Open https://69-text-with-images--vg-volvotrucks-us-rd--netcentric.hlx.page/drafts/tdziezyk/v2/textual-content-with-images in Mobile View
2. Notice the "Explore interior driving environment" button

**Expected result**
Button should be correctly wrapped. Figma design: https://www.figma.com/file/yv0VtHTvMoGPqkv0RAYcVp/PDP?node-id=7499%3A38363&mode=dev

**Actual result**
Button is not correctly wrapped in Mobile view

image

### PR DESCRIPTION
**Pull request by cogniSyb**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #98 #102 #105 

Test URLs:
- Before: https://develop--vg-volvotrucks-us-rd--netcentric.hlx.page/pdp
- After: https://102-refactor-buttons--vg-volvotrucks-us-rd--netcentric.hlx.page/pdp

### CODE REVIEW

File: styles/styles.css [hunk 1]
```diff
@@ -1086,9 +1086,13 @@ main.blue-contract .section.section-with-title p {
   justify-content: center;
   letter-spacing: 1.25px;
   line-height: 100%;
-  padding: 17px 36px;
-  text-align: center;
+  max-width: 21.5em;
```
Comment (kesiah) [inline styles/styles.css:1089]: why is needed?

File: blocks/v2-video/v2-video.js [hunk 2]
```diff
@@ -54,7 +54,7 @@ export default async function decorate(block) {
   content.classList.add(`${blockClass}__content`);
   [...headings].forEach((heading) => heading.classList.add(`${blockClass}__heading`));
   [...ctaButtons].forEach((button) => {
-    button.classList.add(`${blockClass}__button`, 'tertiary');
```
Comment (kesiah) [inline blocks/v2-video/v2-video.js:57]: Hover state is not correct:
image

**Review by kesiah — CHANGES_REQUESTED** [review]

File: styles/styles.css (hunk 1 above)
Comment (kesiah) [reply styles/styles.css:1089]: 😢 
image

**Review by Lakshmishri — APPROVED** [review]

File: styles/styles.css [hunk 3]
```diff
@@ -1086,9 +1086,13 @@ main.blue-contract .section.section-with-title p {
...
+  height: 48px;
+  padding: 0 36px;
+  text-align: left;
+  text-transform: uppercase;
```
Comment (kesiah) [inline styles/styles.css:1093]: It was decided at the beginning of the project that the uppercase will be editor decision. Did this decision change? Now ALL buttons will have uppercase?

File: styles/styles.css [hunk 4]
```diff
@@ -1170,9 +1174,9 @@ main.blue-contract .section.section-with-title p {
 
 .redesign-v2 a.button.tertiary,
 .redesign-v2 button.tertiary {
-  padding: 8px 12px 8px 0;
+  padding: 8px 0;
+  height: 40px;
```
Comment (kesiah) [inline styles/styles.css:1178]: I don't think `height` override is needed.

**Review by kesiah — CHANGES_REQUESTED** [review]

**Review by kesiah — APPROVED** [review]

**Pull request merged**